api/ChannelSplitterNode.json

```diff
@@ -0,0 +1,107 @@
+{
+  "api": {
+    "ChannelSplitterNode": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/ChannelSplitterNode",
+        "support": {
+          "webview_android": {
+            "version_added": true
+          },
+          "chrome": {
+            "version_added": "14"
```

Comment:
Chromes note is missing (all chromes + opera): [1] Starting in Chrome 56, channelCountMode is set to "explicit" and channelCount is fixed to the number of outputs, as per the latest spec.

Reply:
Cool, added.